src/network.rs

```diff
@@ -151,6 +153,12 @@ pub struct Nectar {
     orderbook: orderbook::Orderbook,
     setup_swap: setup_swap::SetupSwap<SetupSwapContext>,
     #[behaviour(ignore)]
+    seed: Seed,
+    #[behaviour(ignore)]
+    key_pair: Keypair,
```

Comment:
Calling this `identity` would make it easier to reason about.

Reply:
I trust you on this one.

Reply:
Would it? Because there are quite a few `identity`s in this module (comit ones).